tests/components/ihc/test_config_flow.py

```diff
@@ -0,0 +1,190 @@
+"""Test the ihc config flow."""
+from unittest.mock import patch
+
+from homeassistant import config_entries
+from homeassistant.components.ihc.const import DOMAIN
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def test_form(hass: HomeAssistant) -> None:
```

Comment:
Please make `mock_setup_entry` a fixture in `conftest.py`

```suggestion
pytestmark = pytest.mark.usefixtures("mock_setup_entry")


async def test_form(hass: HomeAssistant, mock_setup_entry: AsyncMock) -> None:
```